secrets/deployment.yml.example

```diff
@@ -60,6 +61,7 @@ latest_rabbitmq_amqp_survey_response:
 latest_json_secret_keys:
 latest_secure_message_client_secret:
 latest_response_operations_client_secret:
+latest_response_operationssocial_client_secret:
```

Comment:
I know this file isn't used directly, but might be worth changing to: `latest_response_operations_social_client_secret`